src/views/List.jsx

```diff
@@ -84,25 +67,20 @@ export function List({ data, loading }) {
 			// - kind of soon: (between 7 & 30 days until the next purchase)
 			// - soon: (7 days or fewer until the next purchase)
 			if (daysUntilNextPurchase >= 60) {
-				buyingUrgency = 'inactive';
-				colorUrgency = '#878E88';
+				imgUrgency = '../img/bread_styling/inactive-loaf.svg/';
```

Comment:
Nit: update imports here

Reply:
might be echoing but you may need to avoid having the `/` on the end of the file path here.